vg/draw/canvas.go

```diff
@@ -55,6 +55,12 @@ type GlyphStyle struct {
 	// Radius specifies the size of the glyph's radius.
 	Radius vg.Length
 
+	// Width is set, LineColour is used to outline the glyph
```

Comment:
Great suggestion, I would like that idea, except for the fact that `LineStyle` also has `Dashes` and `DashOffs` fields, they would be effectively unused. Im happy to switch the patch over to use `LineStyle` if that is what people prefer.